js/script.js

```diff
@@ -0,0 +1,42 @@
+console.log('hello is this showing up?');
+//The user types in the name of the city
+//The user clicks on the submit button, which carries out the function findCity
+//The function is prevented from going into the default action
+//The function extracts and stores the user input
+//the user may use lowercase, so make sure strings are converted to lowercase.
+//The stored name of the city is then evaluated to a set of conditionals
+function error() {
+  alert('Oops! There is no picture for this city!');
+}
+
+function findCity(event) {
+  event.preventDefault(); 
+  console.log('submitting the form');
+
+  var city = $('#city-type').val();
+  
+    if (city.toLowerCase().toUpperCase() === 'New York'.toLowerCase().toUpperCase() || city.toLowerCase().toUpperCase() === 'New York City'.toLowerCase().toUpperCase() || city.toLowerCase().toUpperCase() === 'NY'.toLowerCase().toUpperCase() || city.toLowerCase().toUpperCase() === 'NYC'.toLowerCase().toUpperCase()) {
```

Comment:
Right idea. Although, there's no need to convert to lowercase and then to uppercase. Just do one or the other.

You also could have saved yourself a bit of typing if you had "normalized" the `city` variable prior to the `if` block.

e.g.

```js
var city = $('#city-type').val();
city = city.toLowerCase(); // now city is always lower cased

// now compare city to the lower case variants of each city
if (city === 'new york city' || city === 'new york' || city === 'nyc' || city === 'ny') {
  // ....
}
```